lib/storage/file.js

```diff
@@ -1019,12 +1019,18 @@ File.prototype.getSignedPolicy = function(options, callback) {
  *     link will expire.
  * @param {string=} options.extensionHeaders - If these headers are used, the
  *     server will check to make sure that the client provides matching values.
+ * @param {string=} options.responseDisposition - If you provide this value, 
+ *     the response-content-disposition parameter of the signed url is set 
+ *     accordingly.
+ * @param {string=} options.responseType - If you provide this value, the 
+ *     response-content-type parameter of the signed url is set accordingly.
  * @param {function=} callback - The callback function.
  *
  * @example
  * file.getSignedUrl({
  *   action: 'read',
- *   expires: Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14) // 2 weeks.
+ *   expires: Math.round(Date.now() / 1000) + (60 * 60 * 24 * 14), // 2 weeks.
+ *   responseDisposition: 'attachment; filename="filename.ext"'
```

Comment:
Yikes. This looks very much like a raw header. Obviously if you think that flexibility is needed then ok, otherwise perhaps maybe the option could be renamed as something more user friendly and descriptive like  `promptSaveAs: 'filename.ext'` where filename.ext is the name of the file they want the browser to prompt them to save as, which is what I understand this option does.